charts/argo-workflows/templates/server/server-deployment.yaml

```diff
@@ -87,6 +87,22 @@ spec:
               {{- end }}
             initialDelaySeconds: 10
             periodSeconds: 20
+          {{- if .Values.server.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: {{ .Values.server.livenessProbe.httpGet.path }}
+              port: {{ .Values.server.livenessProbe.httpGet.port }}
+              {{- if .Values.server.secure }}
+              scheme: HTTPS
```

Comment:
this is only true if https is enabled on the pod. Which I don't think the chart does by default (but upstream does)